setup.py

```diff
@@ -24,7 +24,7 @@
 
 MAJOR = 2
 MINOR = 6
-MICRO = 2
+MICRO = 3
```

Comment:
Technically would could delete the 2.6.2 tag and retag it if we wanted to. After #205 the CI job isn't eagerly publishing sdists anymore so nothing 2.6.2 made it outside of github. But I'm fine to also tag this as 2.6.3